src/Utils/TrackState.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Created by imamudin.naseem on 02/05/16.
+ */
+
+'use strict'
+import t from 'argtoob'
+import {Observable} from 'rx'
+
+export default ({audio$, selectedTrackId$}, id) => {
+  const events$ = audio$.withLatestFrom(selectedTrackId$, t('event', 'id'))
+  return Observable.merge(
+    events$.filter(x => x.event === 'playing' && x.id === id).map('PLAYING'),
+    events$.filter(x => x.event === 'pause' && x.id === id).map('PAUSED'),
+    events$.filter(x => x.event === 'play' && x.id === id).map('LOADING'),
+    events$.filter(x => x.id !== id).map('STOPPED')
+  )
+}
```

Comment:
This function doesn't seem to do much apart from changing the case of the events that are being dispatched :)
Do we really need it? Can't we do something simpler may be?